lib/widgets/new_dm_sheet.dart

```diff
@@ -0,0 +1,331 @@
+          fontSize: 17, height: 22 / 17),
+        border: InputBorder.none));
+  }
+}
+
+class _SelectedUserChip extends StatelessWidget {
+  const _SelectedUserChip({
+    required this.userId
+  });
+
+  final int userId;
+
+  @override
+  Widget build(BuildContext context) {
+    final designVariables = DesignVariables.of(context);
+    final store = PerAccountStoreWidget.of(context);
+    final user = store.getUser(userId)!;
+
+    return Container(
+    decoration: BoxDecoration(color: designVariables.bgMenuButtonSelected, borderRadius: BorderRadius.circular(3)),
+    child: Row(
+      mainAxisSize: MainAxisSize.min,
+      children: [
+        Avatar(userId: userId, size: 22, borderRadius: 3),
+        Padding(
+          padding: const EdgeInsetsDirectional.fromSTEB(5, 3, 4, 3),
+          child: Text(user.fullName,
+            style: TextStyle(
+              fontSize: 16,
+              height: 16 / 16,
+              color: designVariables.labelMenuButton))),
+      ]));
+  }
+}
+
+class _NewDmUserList extends StatelessWidget {
+  const _NewDmUserList({
+    required this.filteredUsers,
+    required this.selectedUserIds,
+    required this.onUserTapped
+  });
+
+  final List<User> filteredUsers;
+  final Set<int> selectedUserIds;
+  final void Function(int userId) onUserTapped;
+
+  @override
+  Widget build(BuildContext context) {
+    final designVariables = DesignVariables.of(context);
+
+    return ListView.builder(
+      padding: EdgeInsets.all(8),
+      itemCount: filteredUsers.length,
+      itemBuilder: (context, index) {
+        final user = filteredUsers[index];
+        final isSelected = selectedUserIds.contains(user.userId);
+
+        return InkWell(
+          onTap: () => onUserTapped(user.userId),
+          child: Container(
+            padding: const EdgeInsets.symmetric(vertical: 2),
+            decoration: isSelected ? BoxDecoration(
+              color: designVariables.bgMenuButtonSelected,
+              borderRadius: BorderRadius.circular(10),
+            ) : null,
+            child: Row(
+              children: [
+                Padding(
+                  padding: const EdgeInsets.all(8),
+                  child: Icon(
+                    isSelected ? Icons.check_circle : Icons.circle_outlined,
+                    color: isSelected ? designVariables.radioFillSelected : designVariables.radioBorder,
+                    size: 24)),
+                Avatar(userId: user.userId, size: 32, borderRadius: 3),
+                Expanded(
+                  child: Padding(
+                    padding: const EdgeInsetsDirectional.fromSTEB(8, 6.5, 0, 6.5),
+                    child: Text(user.fullName,
+                      style: TextStyle(
+                        fontSize: 17,
+                        height: 19 / 17,
+                        color: designVariables.textMessage)
+                      .merge(weightVariableTextStyle(context, wght: 500))))),
+              ])));
```

Comment:
This group of widgets should have some vertical and right paddings:

![image](https://github.com/user-attachments/assets/b7256c75-9e19-4c19-9cb1-16af22294817)

I see that the leading check_circle icon is the tallest widget in the row by default, but that might not be the case with non-default font setting.